OJS.Workers.ExecutionStrategies/Python/PythonCodeExecuteAgainstUnitTestsExecutionStrategy.cs

```diff
@@ -87,6 +87,13 @@ public class PythonCodeExecuteAgainstUnitTestsExecutionStrategy : PythonExecuteA
             return processExecutionResult;
         }
 
+        protected void WriteTestInCodeFile(string code, string codeSavePath, string testContent)
+        {
+            var codeAndTestText = code + Environment.NewLine + testContent;
+
+            FileHelpers.WriteAllText(codeSavePath, codeAndTestText);
```

Comment:
Since this is new, I think it is a good idea to extract it into a service, so it can be tested someday.
To skip duplicating code:
```
class FileIoService { 
   public void WriteAllText(path, text) {
       FileHelpers.WriteAllText(path, text);
   }
}
```

Reply:
The infrastructure does not support dependency injection and service is redundant at this point.